storage/engine/rocksdb/db.cc

```diff
@@ -115,6 +120,15 @@ DBStatus ToDBStatus(const rocksdb::Status& status) {
   return ToDBString(status.ToString());
 }
 
+DBStatus FmtStatus(const char *fmt, ...) {
```

Comment:
`FmtStatus` is not explicit tested, but we do have tests that exercise the code path that memory allocated by `ToDBString` is freed by `statusToError`. Do you have any thoughts on how to test that the memory was actually freed?